adhocracy-plus/assets/scss/components/_a4-comments.scss

```diff
@@ -124,18 +124,34 @@
 
 .a4-comments__action-bar__btn {
     margin-left: 0.2 * $spacer !important;
+    text-decoration: underline;
 
     @extend .btn--small;
 
     &:first-child {
         margin-left: 0 !important;
     }
+
+    &:hover {
+        text-decoration: none;
+    }
 }
 
 .a4-comments__border--highlighted {
     border-left: 2px solid $brand-primary;
 }
 
+.btn.btn--none.a4-comments__read-btn {
```

Comment:
should these changes be ported over/done in a+ ? they are not really defakts specific, are they?

Reply:
@goapunk don't know, it's never been considered an issue in a+ so would say no, assume maybe because theres less buttons there but you can check with Marcelo?